### DISCUSSION TITLE
Cable trace api not found for front-ports and rear-ports

### DISCUSSION THREAD

**Discussion by 4nqyi** [netbox-community/netbox#9998] [General]
I can reach the UI trace page but not the api for rear-ports and front-ports. 
Searching shows that this was fixed: https://github.com/netbox-community/netbox/issues/4736
We're running  v3.2.5.

![image](https://user-images.githubusercontent.com/11065729/184510702-0105eb2c-937f-400b-acfc-a0a9907f2370.png)

**Comment by candlerb** [comment]
Looking at the self-hosted REST API documentation in 3.2.8 there are .../trace/ endpoints for console-ports, console-server-ports, interfaces, power-feeds, power-outlets, power-ports; but not for front-ports or rear-ports.

I guess this is a regression since #4736.

* 16cdf300 (linked from that issue) introduced `test_trace_frontport` and `test_trace_rearport` test cases
* 5ed61369 moved them into ComponentTraceMixin
* df737371 ("Remove legacy connection_status fields") removed Mixins.ComponentTraceMixin from FrontPortTest and RearPortTest

So from that point onwards, there was no test case coverage for this feature.
